pom.xml

```diff
@@ -19,12 +19,12 @@
 
     <groupId>com.yahoo.sshd</groupId>
     <artifactId>sshd_proxy</artifactId>
-    <version>0.2.18-SNAPSHOT</version>
+    <version>0.2.18</version>
```

Comment:
I think we have to the let maven release plugin to do this by doing "mvn release:prepare" after this pr has been merged which will create the tag with sshd_proxy-0.2.18 so travis can publish to bintray.